03_python.Rmd

```diff
@@ -145,46 +170,71 @@ If you are not able to successfully install Jupyter Notebooks on your own, pleas
+Python modules are extensions to the basic capabilities of Python. You can install 
+modules from the terminal where you call Python. The same process will apply 
+regardless of your operating system.
 
 ### Installing Modules using Conda
 
-If you are using Anaconda, it is recommended you install modules from conda sources to ensure compatibility. First, start a terminal using the Python environment you want to install the module in.
+If you are using Anaconda, it is recommended you install modules from `conda` 
+sources to ensure compatibility. First, start a terminal using the Python 
+environment you want to install the module in.
 
-**(Optional)** If you are using Anaconda, use the Anaconda Navigator to launch a terminal of your chosen environment.
+**(Optional)** If you are using Anaconda, use the Anaconda Navigator to launch 
+a terminal of your chosen environment.
 
 ![](`r pm_conda_1` "Opening a terminal from anaconda")
 
-Once you are in the terminal of your chosen environment, you can install any module you know the name of using `conda install <module name>`.
+Once you are in the terminal of your chosen environment, you can install any 
+module you know the name of using `conda install <module name>`.
 
-![](`r pm_conda_2` "installing numpy for python in the terminal using conda")
+![](`r pm_conda_2` "installing numpy for Python in the terminal using conda")
 
-`conda` will then search its repositories for a module matching the name you provided. Note that if you are looking for a module python said was missing in an error code, it may not be the same name you need to use when installing! You will need to search online if the install fails because it says there is no module of that name.
+`conda` will then search its repositories for a module matching the name you 
+provided. Note that if you are looking for a module Python said was missing in 
+an error code, it may not be the same name you need to use when installing! You 
+will need to search online if the install fails because it says there is no 
+module of that name.
 
 ### Installing Modules using Pip
 
-If you are not using conda, or conda could not find a module, you will need to use `pip` to install modules. First, start a terminal using the Python environment you want to install the module in.
+If you are not using conda, or conda could not find a module, you will need to 
+use `pip` to install modules. First, start a terminal using the Python environment 
+you want to install the module in.
 
-Once you are in the terminal of your chosen environment, you can install any module you know the name of using `pip install <module name>`.
+Once you are in the terminal of your chosen environment, you can install any 
+module you know the name of using `pip install <module name>`.
 
-![](`r pm_pip_1` "installing numpy for python in the terminal using pip")
+![](`r pm_pip_1` "installing numpy for Python in the terminal using pip")
```

Comment:
Is this an example, or do we want people to install numpy? Either way doesn't hurt, but might add preceding language "for example" or "use pip to install the module numpy" as a pointer.

Reply:
I've added language in this segment to clarify that we're using `NumPy` as an example.